public/content/exjobbs/2024_broman_adaptivemesh.mdx

```diff
@@ -0,0 +1,25 @@
+---
+name: "Contextual Visualization of Adaptive Meshes using YT"
+description: "Contextual Visualization of Adaptive Meshes using YT"
+location: "New York City"
+period: "Spring 2024"
+number_of_students: "1-2"
+contact: [ "emmbr26", "alebo68" ]
+
+finished: false
+skills: [ "OpenSpace", "Volume rendering", "C++", "Python" ]
+---
+
+[OpenSpace](https://openspaceproject.com) is an open-source software that aims to visualize our entire known universe in its accurate context.  It is a mature software that can be used by anyone to tell stories about the cosmos and our place in it.
+
+One of the aspects of OpenSpace that is very interesting to astrophysics researchers, but has not received much recent attention is the volume rendering of complex datasets, such as AMR (adative mesh refinement) grids, commonly used in cosmological models explaining the origin of structure in the universe.
+
+Physicists already use freely available tools, such as [yt](https://yt-project.org/), which however lacks the features for immersive display environments, such as dome theaters, or the ability to load different datasets into one consistent scene description.
+
+The goal of this thesis is to devise a method to ingest a rendering created using yt into the OpenSpace framework.  By utilizing the capabilities of yt rather then reimplementing them, we can combine the functionalities of these two softwares use each for what it is best at.  This approach has been previously used to combine a molecular rendering engine VIAMD into OpenSpace ([paper](https://www.diva-portal.org/smash/get/diva2:1754819/FULLTEXT01.pdf)).
```

Comment:
```suggestion
The goal of this thesis is to devise a method to ingest a rendering created using yt into the OpenSpace framework.  By utilizing the capabilities of yt rather than reimplementing them, we can combine the functionalities of these two software and use each for what it is best at.  This approach has been previously used to combine a molecular rendering engine VIAMD into OpenSpace ([paper](https://www.diva-portal.org/smash/get/diva2:1754819/FULLTEXT01.pdf)).
```